modules/restoring-red-hat-quay-standalone.adoc

```diff
@@ -0,0 +1,218 @@
+:_content-type: PROCEDURE
+[[restoring-red-hat-quay-standalone]]
+= Restoring {productname} on standalone deployments 
+
+This procedure describes how to restore {productname} on standalone deployments. 
+
+.Prerequisites 
+
+* You have backed up your {productname} deployment. 
+
+.Procedure 
+
+. Create a new directory that will bind-mount to `/conf/stack` inside of the {productname} container:
++
+[source,terminal]
+----
+$ mkdir /opt/new-quay-install
+----
+
+. Copy the contents of your temporary backup directory created in xref:backing-up-red-hat-quay-standalone[Backing up {productname} on standalone deployments] to the `new-quay-install1` directory created in Step 1:
++
+[source,terminal]
+----
+$ cp /tmp/quay-backup/quay-backup.tar.gz /opt/new-quay-install/
+----
+
+. Change into the `new-quay-install` directory by entering the following command:
++
+[source,terminal]
+----
+$ cd /opt/new-quay-install/
+----
+
+. Extract the contents of your {productname} directory: 
++
+[source,terminal]
+----
+$ tar xvf /tmp/quay-backup/quay-backup.tar.gz *
+----
++
+Example output:
++
+----
+config.yaml
+config.yaml.bak
+extra_ca_certs/
+extra_ca_certs/ca.crt
+ssl.cert
+ssl.key
+----
+
+. Recall the `DB_URI` from your backed-up `config.yaml` file by entering the following command: 
++
+[source,terminal]
+----
+$ grep DB_URI config.yaml
+----
++
+Example output:
++
+[source,yaml]
+----
+postgresql://<username>:test123@172.24.10.50/quay
+----
+
+. Run the following command to enter the PostgreSQL database server: 
++
+[source,terminal]
+----
+$ sudo postgres 
+----
+
+. Enter psql and create a new database, for example, `example_restore_registry_quay_database`, by entering the following command:
++
+[source,terminal]
+----
+$ psql "host=<user_name>.postgres.database.azure.com port=5432 dbname=postgres user=w..y password=A...2"
+postgres=> CREATE DATABASE example_restore_registry_quay_database;
+----
++
+Example output:
++
+----
+CREATE DATABASE
+----
+
+. Connect to the database by running the following command:
++
+[source,terminal]
+----
+postgres=# \c "example-restore-registry-quay-database";
+----
++
+Example output:
++
+[source,terminal]
+----
+You are now connected to database "example-restore-registry-quay-database" as user "postgres".
+----
+
+. Create a `pg_trmg` extension of your Quay database by running the following command:
++
+[source,terminal]
+----
+example_restore_registry_quay_database=> CREATE EXTENSION IF NOT EXISTS pg_trgm;
+----
++
+Example output:
++
+[source,terminal]
+----
+CREATE EXTENSION
+----
+
+. Exit the postgres CLI by entering the following command: 
++
+[source,terminal]
+----
+\q
+----
+
+. Set the password for your PostgreSQL deployment by running the following command:
```

Comment:
What we do here is to import  database backup to the new database, not to set the password.